schema/schema.json

```diff
@@ -0,0 +1,122 @@
+{
+    "description": "Schema for OpenContainer bundle configuration file",
```

Comment:
On Wed, Mar 09, 2016 at 10:22:17AM -0800, Vincent Batts wrote:
> > @@ -0,0 +1,122 @@
> > +{
> > +    "description": "Schema for OpenContainer bundle configuration file",
> 
> i'm not keen on using `config.json`, even though this file is in a
> `./schema/` directory. At best, perhaps `config-schema.json`

I don't see the point of including ‘schema’ in the filename (it's
already in the full path), but ‘config-schema.json’ still lets us add
schemas for state.json, etc., so it addresses my practical concern.